yarn-project/world-state/src/world-state-db/index.ts

```diff
@@ -65,7 +65,7 @@ type WithIncludeUncommitted<F> = F extends (...args: [...infer Rest]) => infer R
 /**
  * Defines the names of the setters on an append only set of Merkle Trees.
```

Comment:
I guess we want to update the comment here since it's no longer an append only set of trees.